### PR TITLE
User can view a trip's main summary

### PR DESCRIPTION
**Pull request by jennydtran**
Jan-02-2021 13-15-23

### CODE REVIEW

File: client/components/modal-addtrip.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    let month = today.getMonth() + 1;
+    let day = today.getDate() + 1;
+    const year = today.getFullYear();
+    if (month < 10) {
+      month = '0' + month.toString();
+    }
+    if (day < 10) {
+      day = '0' + day.toString();
+    }
+    const maxDate = `${year}-${month}-${day}`;
```
Comment (thebearingedge) [inline client/components/modal-addtrip.jsx:81]: I noticed this is in another component too. I recommend moving a helper function into its own file so you can import it into each component file.

File: client/pages/trip.jsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    super(props);
+    this.state = {
+      currentTrip: null,
+      todoList: [],
+      travelersList: [],
+      transportationList: [],
+      accommodationList: [],
+      activitiesList: [],
+      placesList: [],
+      packingList: []
```
Comment (thebearingedge) [inline client/pages/trip.jsx:15]: I'm not too sure that these are for.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Just some minor changes.

File: client/pages/trip.jsx (hunk 2 above)
Comment (jennydtran) [reply client/pages/trip.jsx:15]: For this part, I kind of... thought ahead. It's not needed now.

File: client/app.jsx [hunk 3]
```diff
@@ -1,8 +1,45 @@
...
+
+  renderPage() {
+    const { route } = this.state;
+
+    if (route.path === '') {
+      return <Home />;
+    }
+    if (route.path === 'trips') {
+      const tripId = route.params.get('tripId');
+      // this.getSingleTrip(tripId);
```
[the remark this answers is not in the document]
Comment (jennydtran) [reply client/app.jsx:33]: Forgot to take that out...

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**